client/src/components/NavBar/Notifications/NotificationCenter/NotificationCenter.tsx

```diff
@@ -53,16 +52,16 @@ const NotificationCenter = ({ loggedInUser, notifications }: NotificationProps):
         }}
       >
         {notifications
-          .filter((item: INotificationItem) => item.user === loggedInUser.email)
+          .filter((item: INotificationItem) => item.user === loggedInUser?.email)
           .map((item: INotificationItem) => {
             return (
               <NotificationItem
                 key={item.title}
                 type={item.type}
                 title={item.title}
                 description={item.description}
-                read={item.read}
-                link={item.link}
+                read={Boolean(item.read)}
```

Comment:
Are the read booleans strings? Strings are truthy in JavaScript. `Boolean('false')` would still come up as true.

Reply:
If they are strings you could do a comparison `item.read === 'true'`

Reply:
The problem here was that read is either boolean or null. Typescript was complaining about the null. Boolean(read) is guaranteed to be true or false.

Reply:
But I will check why I was getting this boolean or null in the first place. It seems weird now that I think about it.

Reply:
Yeah, read should just either be false or true not true or null. 👍🏻